protocol/daemons/liquidation/client/client.go

```diff
@@ -192,9 +156,73 @@ func GetAllSubaccounts(
 			break
 		}
 	}
+
+	telemetry.ModuleSetGauge(
+		metrics.LiquidationDaemon,
+		float32(len(subaccounts)),
+		metrics.GetAllSubaccounts,
+		metrics.Count,
+	)
+
 	return subaccounts, nil
 }
 
+// GetLiquidatableSubaccountIds verifies collateralization statuses of subaccounts with
+// at least one open position and returns a list of unique and potentially liquidatable subaccount ids.
```

Comment:
Where does this function dedupe subaccount IDs?

Reply:
why do we need to dedupe? there won't be any duplicates, right?